src/index.js

```diff
@@ -90,17 +93,20 @@ function createSolutionInfos() {
   ));
 }
 
-function processNext(next, config) {
-  const branch = config.submodules[next];
-  console.log(`Processing ${next}`);
+function processSubmodule(name, config) {
+  const branch = config.submodules[name];
 
-  const [owner, repo] = next.split('/');
+  const [owner, repo] = name.split('/');
   const submodule = {
+    _logs: [],
     owner,
     repo,
     branch,
-    remoteUrl: `git@git:${owner}/${repo}.git`
+    remoteUrl: `git@git:${owner}/${repo}.git`,
+    log(message, indent=0) { this._logs = this._logs.concat(message.split(/\r?\n/).map((line) => { return Array(indent + 1).join(' ') + line; })); },
```

Comment:
That is only available in ES6 and if you try it in a node repl it is not defined.

Also, for indent=0, you would still get one space.

With Array(0 + 1).join(' ') you get an empty string.  For indent=3, you get an array with 3 elements which are joined to give two spaces.